### DISCUSSION TITLE
[Example] 공식 샘플 월드 Official World Samples

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#615] [2. Official Samples]
안녕하세요, 제페토 에반젤리스트입니다.
공식 샘플 예제 월드가 업데이트 되었습니다. 

### [ZEPETO World Sample]
https://github.com/naverz/zepeto-world-sample

제페토 월드 개발에 도움을 드리고자 샘플 예제를 난이도 별로 제작하게 되었습니다
예제는 추가로 업데이트 될 예정이며, 각 예제에 대한 상세한 강의 영상도 준비 중이오니 조금만 기다려 주세요😊
<br>
<br> 
### [챕터 소개]

- Chapter 1 : 오브젝트 이동과 텔레포트로 완성하는 점프맵
> Teleport, GameObject Movement, GameObject Rotator

https://user-images.githubusercontent.com/100118650/163148033-8fc2760d-6598-4f4e-a864-4a910b2eaf09.mov

- Chapter 1 : 강의 영상
> https://studio.zepeto.me/guides/world-tutorial-chapter1

<br>
<br> 

- Chapter 2 : 캐릭터 이동 속도와 점프 제어를 이용한 아이템 먹기 월드

> Item Get, UI Controller, Jump Power, Run Speed Up

https://user-images.githubusercontent.com/100118650/163150138-f010af3d-d211-4864-ba01-5da97c31f8fb.mov

<br>
<br> 

- Chapter 3 : 상호작용을 이용한 간단한 랜덤 월드

> Interaction Button, Gesture

https://user-images.githubusercontent.com/100118650/163148471-3878c10a-e6b5-482e-b21c-d6939aa452bc.mov

<br>
<br> 

### [폴더 소개]

- Scenes : 폴더 내의 Chapter Scene 파일을 더블클릭해서 Play 할 수 있습니다.
- Scripts : 폴더 내의 스크립트를 이용해서 원하는 기능을 손쉽게 구현할 수 있습니다. 원하는 기능을 복사 붙여넣기로 가져오신 후 숫자 정도만 변경하면 되는 난이도로 작성하였습니다. 
- Materials : 각 챕터에 사용된 메테리얼 파일입니다.
- Prefabs : 각 챕터에 사용된 프리팹 파일입니다.
- Resources : 제페토에서 공식적으로 제공하는 맵, BGM, Skybox 등 각종 리소스 파일입니다. 월드 제작시 자유롭게 사용이 가능합니다.

<br>
<br> 
<hr/>

Hello, I'm ZEPETO Evangelist.
The official sample example world has been updated.

### [ZEPETO World Sample]
https://github.com/naverz/zepeto-world-sample

To help you develop ZEPETO World, we've made sample examples by level of difficulty
The examples will be updated further, and we are also preparing detailed lecture videos for each example, so please wait a little longer😊

### [Chapter introduction]

- Chapter 1: Jump map completed by moving game objects and teleporting
> Teleport, GameObject Movement, GameObject Rotator

- Chapter 2 : Item eating game using character movement speed and jump control
> Item Get, UI Controller, Jump Power, Run Speed Up

- Chapter 3: Simple Random Game Using Interaction
> Interaction Button, Gesture

### [Introducing the folder]

- Scenes: You can double-click the Chapter Scene file in the folder and play it.
- Scripts: You can easily implement the functions you want using the scripts within the folder. After importing the function you want to copy paste, you can only change the number to a difficulty level. 
- Materials : Materials file used in each chapter.
- Prefabs : Prefab files used in each chapter.
- Resources: Map, BGM, Skybox, etc. are officially provided by ZEPETO. You can use it freely when making the world.

**Comment by smilejsu82** [comment]
wow! 👍

**Reply by x4cf** [reply]
Wow too!

**Reply by mayalhc** [reply]
오~좋은 자료 감사합니다
